htdocs/router.php

```diff
@@ -10,13 +10,14 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-
-
 $url     = ltrim($_SERVER['REQUEST_URI'], "/");
 $request = $_SERVER['REQUEST_URI'];
 
 if ($request != '/'
-    && file_exists(__DIR__ . $request)
+    && (
+        file_exists(__DIR__ . $request)
+        || file_exists(__DIR__ . "/" . $urlpath)
```

Comment:
this should probably just be the $urlpath line, but I didn't want to risk breaking anything else this late in the release cycle, so I just kept both.